### PR TITLE
Improve invasion description to account for manual and automatic ship…

### PR DESCRIPTION
**Pull request by Oberlus**
… selection

### CODE REVIEW

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -9362,6 +9362,8 @@ PC_TROOPS_DESC
 
 To successfully invade a planet, the total number of [[metertype METER_TROOPS]] from all invading ships must exceed the total defending (ties are awarded to the defender).
 
+When invading a planet, the troop ships selected in the fleet window will be used for the invasion. If no troop ships were selected, enough troop ships will be automatically selected.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: remove "enough" as there might not be that many available

Comment (geoffthemedio) [reply default/stringtables/en.txt]: were -> are

Comment (geoffthemedio) [reply default/stringtables/en.txt]: maybe add "from the available ships and based on the number of defending troops".

Comment (Oberlus) [reply default/stringtables/en.txt]: Fixed as per your suggestions.

**Pull request merged**